pkg/config/config_test.go

```diff
@@ -39,6 +39,7 @@ log:
    file: agent.log
    forward: true
    level: debug
+   logging_retry_limit: 10
```

Comment:
The `log` object is for the agent logging configuration. `logging_retry_limit` should not be inside this object (as `LoggingHomeDir`, `LoggingBinDir` ...)

Reply:
😮‍💨 oh right... that's why it was with "bad indentation" reverting that change, also fixed the tests that were wrong.